src/math/PolylineIntersector.cpp

```diff
@@ -87,7 +87,12 @@ bool PolylineIntersector::intersectsSpan(const QLineF& normal, const QLineF& spa
 QPointF PolylineIntersector::intersectWithSegment(const QLineF& line, int segment) const {
   const QLineF segLine(m_polyline[segment], m_polyline[segment + 1]);
   QPointF intersection;
-  if (line.intersects(segLine, &intersection) == QLineF::NoIntersection) {
+#if QT_VERSION_MAJOR  == 5
```

Comment:
Shouldn't we check here with ` #if QT_VERSION_MAJOR == 5 and QT_VERSION_MINOR < 14` too?